assets/lokomotive-kubernetes/bootkube/resources/charts/kube-apiserver/templates/kube-apiserver.yaml

```diff
@@ -14,6 +14,12 @@
 6443
 {{- end -}}
 {{ end }}
+{{- define "token" -}}
+{{ include (print $.Template.BasePath "/kube-apiserver-secret.yaml") . | sha256sum }}
```

Comment:
I assume this is the rendered file, right?

Reply:
Yes, this is already a template with values included. The pattern is from https://helm.sh/docs/howto/charts_tips_and_tricks/ "https://helm.sh/docs/howto/charts_tips_and_tricks/" section.

Reply:
cool

Reply:
Ditto inline comment.